### PR TITLE
Add translations to all aria-labels

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

**Comment by leomendoza123** [comment]
@DanielPalafox @amontenegro To easily test this PR please let's review and merge this codependency first https://github.com/ORCID/orcid-angular/pull/197 so is easier to read the changes specific to this PR

**Comment by leomendoza123** [comment]
[Aria labels not translation in orcid-angular (2)](https://trello.com/c/a5J3e9wc/6710-aria-labels-not-translation-in-orcid-angular-2)

File: src/app/institutional/pages/institutional/institutional.component.html [hunk 1]
```diff
@@ -1,102 +1,113 @@
...
-              i18n="@@institutional.continue"
-            >
-              CONTINUE
-            </button>
-          </div>
-        </div>
-      </form>
-    </mat-card-content>
-  </mat-card>
+<main>
```
Comment (DanielPalafox) [inline src/app/institutional/pages/institutional/institutional.component.html]: id attribute is missing

File: src/locale/properties/search/search.en.properties [hunk 2]
```diff
@@ -14,3 +14,4 @@ ngOrcid.search.keyword=Keyword
 ngOrcid.search.otherNameFields=Also search other name fields
 ngOrcid.search.search=SEARCH
 ngOrcid.search.orcidIdPatterError=Given value doesn't match ORCID iD pattern
+ngOrcid.search.ariaLabelSearchResults=Search Results
```
Comment (DanielPalafox) [inline src/locale/properties/search/search.en.properties:17]: properties missing in xx, ltr and rtl properties files

File: src/app/institutional/pages/institutional/institutional.component.html (hunk 1 above)
Comment (leomendoza123) [reply src/app/institutional/pages/institutional/institutional.component.html]: Fix with merge to master

File: src/locale/properties/search/search.en.properties (hunk 2 above)
Comment (leomendoza123) [reply src/locale/properties/search/search.en.properties:17]: oops cloned some missing things, thanks  👍

**Pull request merged**